.changeset/green-cheetahs-cover.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core-app-api': minor
+---
+
+RouteTracker now forwards the route parameters as attributes of the navigate event
```

Comment:
Would recommend not mentioning `RouteTracker`, since it's kind of an internal implementation detail.  But the rest of this is great info!

Reply:
good catch